bin/cli.js

```diff
@@ -281,8 +281,18 @@ program
     ""
   )
   .option(
-    "--html",
-    "Get a html file of the template generated with the Liquid Test information (optional)",
+    "--html-input",
+    "Get a static html of the input-view of the template generated with the Liquid Test data (optional)",
+    false
+  )
+  .option(
+    "--html-preview",
+    "Get a static html of the export-view of the template generated with the Liquid Test data (optional)",
+    false
+  )
+  .option(
+    "--skip-results",
```

Comment:
You are not using `skip-results`

Reply:
`--skip-results` will be quite similar to `--status` (which only displays PASSED/FAILED status back). 
Do we need this slight variant?

Reply:
I've renamed it to `--html-only`; I don't think it will be similar though, cause status gives ONLY final liquid test output, while this will give ONLY preview output.